### ISSUE TITLE
VS Code 1.63.0 makes IW copy whole cell into notebook

### ISSUE THREAD

**Issue by janosh** [microsoft/vscode-jupyter#8504]
It appears updating to VS Code 1.63.0 changes the IW behavior. Before when running a cell, it would only show the first line of the cell above the output in the notebook. Now the whole cell's content is included in the notebook:

![Screen Shot 2021-12-09 at 18 25 20](https://user-images.githubusercontent.com/30958850/145454169-93b72435-ca8f-4f5c-a504-499416450e76.png)

**Comment by IanMatthewHuff** [comment]
Yeah, sorry about the regression here. We have a duplicate filed for this issue here to track so I'll close this one as a duplicate.
https://github.com/microsoft/vscode-jupyter/issues/8203
